superset-frontend/src/theme/index.ts

```diff
@@ -181,6 +181,7 @@ const baseConfig: ThemeConfig = {
     },
     Popover: {
       colorBgElevated: supersetTheme.colors.grayscale.light5,
+      zIndexPopup: supersetTheme.zIndex.max,
```

Comment:
We should not be touching on this file any longer as it will likely be removed when the theming PR will be merged https://github.com/apache/superset/pull/31590 and just rely on default. If default isn't working right now, we can keep the custom z-index there was there before and add a TODO that we'll need to remove it

Reply:
This is still the same. Have you acknowledged this comment?